src/PKSim.Core/Services/ParameterTask.cs

```diff
@@ -330,10 +330,18 @@ public ICommand SetParameterValue(IParameter parameter, double value, bool shoul
 
       private IPKSimCommand commandForRelativeExpressionParameter(IParameter parameter, double value)
       {
-         if (!parameter.IsExpressionNorm())
-            return new SetRelativeExpressionInSimulationAndNormalizedCommand(parameter, value);
+         //Normalized expression parameters are readonly and will be updated when setting relative expression
+         if (parameter.IsExpressionNorm())
+            return new PKSimEmptyCommand();
 
-         return new SetRelativeExpressionFromNormalizedCommand(parameter, value);
+         var expressionContainer = parameter.ParentContainer;
+         switch (expressionContainer.ParentContainer)
```

Comment:
This was also wrong somehow. Not crucial at that stage but the individual in the simulation was not update as an invididual/ As such the value of the internal building block (the normalized value in fact) could be out of sync. Probably not an issue anyways but well might as well be right